osu.Game/Overlays/NowPlayingOverlay.cs

```diff
@@ -469,5 +471,98 @@ protected override void OnHoverLost(HoverLostEvent e)
                 base.OnHoverLost(e);
             }
         }
+
+        private partial class ScrollingTextContainer : CompositeDrawable
+        {
+            private const float initial_move_delay = 1000;
+            private const float pixels_per_second = 50;
+
+            private LocalisableString text;
+            private OsuSpriteText mainSpriteText = null!;
+            private OsuSpriteText fillerSpriteText = null!;
+
+            public LocalisableString Text
+            {
+                get => text;
+                set
+                {
+                    text = value;
+                    Schedule(updateText);
```

Comment:
why schedule?

Reply:
`InternalChild` is set in BDL, so the drawables are not instantiated yet at this point so it needs the scheduling.

Thought setting `Child` in BDL is the rule of thumb, but sometimes it's set in the constructor (even recent code). I guess for this case (private nested class and these properties), it's better to put in constructor?

Reply:
Usually that sort of thing is handled by `IsLoaded` checks, not scheduling.